coretex/cli/modules/node.py

```diff
@@ -353,7 +353,7 @@ def configureNode(config: Dict[str, Any], verbose: bool) -> None:
         config["image"] += f":latest-{tag}"
 
     config["storagePath"] = config_defaults.DEFAULT_STORAGE_PATH
-    config["nodeRam"] = int(min(ramLimit, config_defaults.DEFAULT_RAM_MEMORY))
+    config["nodeRam"] = int(min(config_defaults.MINIMUM_RAM_MEMORY, config_defaults.DEFAULT_RAM_MEMORY))
```

Comment:
RAM stands for "Random Access Memory", there is no need to have a "_MEMORY" suffix in names which have "RAM" as well.